File: python/arrays/RemoveDuplicates.py
#sorted array
#TWO_POINTER Approach
#gets number of unique elements

nums = [1,1,2,2,3,4,4]

def RemoveDuplicates(nums):
    unique_index = 0
    for i in range(len(nums)):
        if nums[unique_index] != nums[i]:
            unique_index += 1
            nums[unique_index] = nums[i]
    return unique_index + 1
    
print(RemoveDuplicates(nums))

File: Javascript/array/RemoveDuplicates.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    let u_index = 0
    for(let i = 0; i < nums.length; i++ ){
        if( nums[u_index] != nums[i]){
            u_index++;
            nums[u_index] = nums[i];
        }
    } return (u_index+1)
};